### PR TITLE
Add recipe for python-switch-quotes

### PR DESCRIPTION
**Pull request by werehuman**
### Brief summary of what the package does

Cycle between apostrophes and quotes in python strings.

### Direct link to the package repository

https://github.com/werehuman/python-switch-quotes

### Your association with the package

Maintainer.

### Relevant communications with the upstream package maintainer

***None needed***

### Checklist

- [x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [x] I've used [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] I've built and installed the package using the instructions in the [README](https://github.com/melpa/melpa/blob/master/README.md)

### CODE REVIEW

**Comment by syohex** [comment]
I have sent PR about minimum Emacs version.
- https://github.com/werehuman/python-switch-quotes/pull/1

File: recipes/python-switch-quotes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(python-switch-quotes :fetcher github
+                      :repo "werehuman/python-switch-quotes"
+                      :files ("python-switch-quotes.el"
```
Comment (syohex) [inline recipes/python-switch-quotes]:
```lisp
("*.el" "*.el.in" "dir"
 "*.info" "*.texi" "*.texinfo"
 "doc/dir" "doc/*.info" "doc/*.texi" "doc/*.texinfo"
 (:exclude ".dir-locals.el" "test.el" "tests.el" "*-test.el" "*-tests.el"))
```

Could you rename or move test file name and remove `:files` attribute ? Current test file name is not matched against default `:exclude` pattern(above snippet).
- Renaming: `test-python-switch-quotes.el` -> `python-switch-quotes-test.el`
- Moving: `test-python-switch-quotes.el` -> `test/test-python-switch-quotes.el`

And I suppose README.md is not necessary. It is enough by package comment and docstring.

**Comment by syohex** [comment]
Thanks for fixing.

This package assign key binding(`C-c '`) only by loading. This may override user key configuration even if user does not use this package. I propose that

- Remove its code(and users assign key manually)
- Provide minor-mode(default mode map uses `C-c '` key) or setup function

How about ?

**Comment by werehuman** [comment]
I think that the first variant is better. https://github.com/werehuman/python-switch-quotes/commit/ee36b435070cf19d2156c161d57fc18ed73684ac

**Comment by syohex** [comment]
Thanks!!

**Pull request merged**